docs/application/native/guides/internationalization/i18n.md

```diff
@@ -2351,6 +2357,10 @@ Since Tizen 3.0, you can use the Format API (in [mobile](../../api/mobile/latest
    i18n_format_get_locale(format, type, &language, &country);
    /* For example, language is "en" and country is "US" */
    ```
+   > **Note**  
+   > This function is deprecated from Tizen 5.0. Instead. you can use `i18n_format_get_locale_id()`,
+   > which will return the locale ID parameter. For example for language "en" and country "US", functions
+   > will return "en_US" string.
```

Comment:
Instead, use `i18n_format_get_locale_id()`, which will return the locale ID parameter. For example for language "en" and country "US", this function will return "en_US" string.